docs/quickstart.rst

```diff
@@ -1,17 +1,11 @@
 Quickstart
 ==========
 
-Installation
-------------
-
-aiodynamo requires Python 3.7 or higher.
-
-``pip install aiodynamo``
-
 
 Example
 -------
 
+``pip install aiodynamo[aiohttp]``
```

Comment:
Do we recomment aiohttp or httpx?
I guess aiohttp is more... stable? but httpx is a bit faster? 🤔

Reply:
I don't think we need to make a recommendation.